ext/parser/bq_parser.go

```diff
@@ -49,10 +59,16 @@ func ParseTopLevelUpstreamsFromQuery(query string) []bigquery.ResourceURN {
 			ignoreUpstreamIdx, projectIdx, datasetIdx, nameIdx = 5, 6, 7, 8
 		case "with":
 			ignoreUpstreamIdx, projectIdx, datasetIdx, nameIdx = 9, 10, 11, 12
-		case "view":
```

Comment:
any reason why we are removing `view`

Reply:
the case is already covered by this regex https://github.com/goto/optimus/pull/164/files#diff-ce0076ac87231740fec314d4c04407b96410d666a61fcd3df70cf75a24175f78R33